sr_robot_commander/test/test_robot_commander.py

```diff
@@ -0,0 +1,676 @@
+        self.assertEqual(time_test_value, self.robot_commander._move_group_commander.get_planning_time())
+
+    def test_set_num_planning_attempts(self):
+        raised = False
+        try:
+            self.robot_commander.set_num_planning_attempts(3)
+        except (NameError, TypeError):
+            raised = True
+        self.assertFalse(raised)
+
+    def test_get_end_effector_pose_from_state(self):
+        rs = RobotState()
+
+        for key, value in RA_HOME_ANGLES.items():
+            rs.joint_state.name.append(key)
+            rs.joint_state.position.append(value)
+
+        pose = self.robot_commander.get_end_effector_pose_from_state(rs)
+        self.assertIsInstance(pose, PoseStamped)
+
+    def test_get_planning_frame(self):
+        raised = False
+        try:
+            self.robot_commander.get_planning_frame()
+        except (NameError, TypeError):
+            raised = True
+        self.assertFalse(raised)
+
+    def test_set_and_get_pose_reference_frame(self):
+        reference_frame = "world"
+        self.robot_commander._move_group_commander.set_pose_reference_frame(reference_frame)
+        self.assertEqual(self.robot_commander._move_group_commander.get_pose_reference_frame(), reference_frame)
+
+    def test_get_group_name(self):
+        self.assertEqual(self.robot_commander._name, self.robot_commander.get_group_name())
+
+    def test_refresh_named_targets(self):
+        self.robot_commander.refresh_named_targets()
+        condition_1 = type(self.robot_commander._srdf_names) == list
+        condition_2 = type(self.robot_commander._warehouse_names) == list
+        self.assertTrue(condition_1 and condition_2)
+
+    def test_set_max_velocity_scaling_factor__range_ok(self):
+        self.robot_commander.set_max_velocity_scaling_factor(0.2)
+
+    def test_set_max_velocity_scaling_factor__range_not_ok(self):
+        self.assertRaises(MoveItCommanderException, self.robot_commander.set_max_velocity_scaling_factor, 3)
+
+    def test_set_max_acceleration_scaling_factor__range_ok(self):
+        self.robot_commander.set_max_acceleration_scaling_factor(0.2)
+
+    def test_set_max_acceleration_scaling_factor__range_not_ok(self):
+        self.assertRaises(MoveItCommanderException, self.robot_commander.set_max_acceleration_scaling_factor, 3)
+
+    def test_allow_looking(self):
+        self.robot_commander.allow_looking(True)
+
+    def test_allow_replanning(self):
+        self.robot_commander.allow_replanning(True)
+
+    def test_plan_to_joint_value_target(self):
+        self.reset_to_home()
+        plan = self.robot_commander.plan_to_joint_value_target(RA_EXAMPLE_TARGET, angle_degrees=False,
+                                                               custom_start_state=None)
+        last_point = list(plan.joint_trajectory.points[-1].positions)
+
+        for i in range(0, len(last_point)):
+            last_point[i] = round(last_point[i], 2)
+            if last_point[i] == -0.00:
```

Comment:
why do this? also, if you need to change to positive just use abs

Reply:
Changed